apps/docs-app/docs/features/data-fetching/server-side-data-fetching.md

```diff
@@ -0,0 +1,86 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Server-Side Data Fetching
+
+Analog supports fetching data from the server before loading a page. This can be achieved by defining an async `load` function in `.server.ts` file of the page.
+
+## Setting the Public Base URL
+
+Analog requires the public base URL for creating the server-side data fetching. Set an environment variable, using a `.env` file to define the public base URL.
+
+```
+// .env
+VITE_ANALOG_PUBLIC_BASE_URL="http://localhost:5173"
+```
+
+## Loading Data
+
+To load the data from the server, create a `.server.ts` file that contains the async `load` function alongside the `.page.ts` file.
+
+```ts
+// src/app/pages/index.server.ts
+import { PageServerLoad } from '@analogjs/router';
+
+export const load = async ({
+  params, // params/queryParams from the request
+  req, // H3 Request
+  res, // H3 Response handler
+  fetch, // internal fetch for direct API calls,
+  event, // full request event
+}: PageServerLoad) => {
+  return {
+    loaded: true,
+  };
+};
+```
+
+## Injecting Data
```

Comment:
```suggestion
## Injecting Data

Accessing the data fetched on the server can be done using the `injectLoad` function provided by `@analogjs/router`. This can be used with Component Inputs and Component Input Bindings provided in the Angular Router configuration.
```

Reply:
I separated this section based on your comment. Could you please check the `Injecting the Data` section?